packages/ui-components/src/declaration.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.scss" {
+  const content: { [className: string]: string };
```

Comment:
I'm just getting initiated to CSS modules as I review this...for the sake of my understanding: this is what enables the access-CSS-class-as-object-property functionality of CSS modules, correct?

Reply:
I think so (not exactly sure what you mean). This is what keeps Typescript from complaining about 

```import styles from "path/to/some.module.scss"```

Without it Typescript doesn't know what to do with this and throws an error like `Cannot find module 'some.module.scss'. ts(2307)` It seemed to flake out a little on my yesterday (probably because I was _futzing_ around with type checking), so if you see that error this, this is where the problem will be resolved.

Reply:
Sorry that was poorly worded...I meant that it enabled the module behavior in the sense that it Typescript recognized it as such.